testnet-contracts/contracts/SafeMath.sol

```diff
@@ -0,0 +1,65 @@
+pragma solidity ^0.5.0;
```

Comment:
does this file need to be here, rather than just imported from openzep?

Reply:
`yarn peggy:abi` uses solc to generate the updated contract abi for ebrelayer. It throws an error when the OpenZeppelin files are imported from node_modules:
<img width="1073" alt="Screen Shot 2019-09-15 at 5 33 53 PM" src="https://user-images.githubusercontent.com/15370712/64929811-a22c4580-d7df-11e9-86b1-7d7f890c1c6d.png">

I agree that the best practice here is to import from OpenZeppelin instead of cloning the contracts, we can potentially update solc's configuration so that node_modules is an allowed import directory.

Reply:
does truffle compile not handle this properly?

Reply:
@denalimarsh bump ^?

Reply:
https://github.com/swishlabsco/peggy/pull/2/commits/527279c23994a5a13546f05b2f87caedcd598c77